### ISSUE TITLE
Standalone migrations are not functional when installed with rpg

### ISSUE THREAD

**Issue by kagminjeong** [thuss/standalone-migrations#17]
I am trying to use standalone migrations without rubygems, installing them with rpg (https://github.com/rtomayko/rpg).

When running migrate task I get the following error:

rake aborted!
no such file to load -- /usr/local/lib/ruby/vendor_ruby/vendor/migration_helpers/init

It seems that rpg copies the contents of lib directory to /usr/local/lib/ruby/vendor_ruby, and does not copy vendor directory anywhere. Thus that particular require fails - vendor/migration_helpers does not even exist to be required.

The natural solution in my opinion is to require 'migration_helpers', and if that fails, continuing with the current code path.

There are however multiple issues with the migration_helpers package that would need to be addressed in order for this to work:
1. It extends activerecord classes in init.rb which would not be installed by rpg either (only files in lib get copied).
2. There seems to be no gem package of migration_helpers.
3. The file in lib is called migration_helper.rb which is inconsistent with package name (migration_helpers).
4. The module is called MigrationConstraintHelpers which is inconsistent with package name as well.

The solutions to these issues seem doable but would involve a fair amount of changes. I intend to (attempt to) upstream them but decided to open this ticket first since ultimately standalone migrations is the package I'm trying to get working.

To summarize, changes to standalone migrations that I expect to appear here eventually are:
1. Change in how migration_helpers is required;
2. Changes to migration_helpers, hopefully taken from upstream.
